h2o-automl/src/main/java/ai/h2o/automl/AutoMLBuildSpec.java

```diff
@@ -55,13 +59,95 @@ public AutoMLBuildControl() {
     public String export_checkpoints_dir = null;
   }
 
+  public static final class AutoMLStoppingCriteria extends Iced {
+
+    public static final int AUTO_STOPPING_TOLERANCE = -1;
+
+    private RandomDiscreteValueSearchCriteria _searchCriteria;
+    private double max_runtime_secs_per_model = 0;
+
+    public AutoMLStoppingCriteria() {
+      super();
+      _searchCriteria = new RandomDiscreteValueSearchCriteria();
+    }
+
+    public double max_runtime_secs_per_model() {
+      return max_runtime_secs_per_model;
+    }
+
+    public void set_max_runtime_secs_per_model(double max_runtime_secs_per_model) {
+      this.max_runtime_secs_per_model = max_runtime_secs_per_model;
+    }
+
+    public long seed() {
+      return _searchCriteria.seed();
+    }
+
+    public int max_models() {
+      return _searchCriteria.max_models();
+    }
+
+    public double max_runtime_secs() {
+      return _searchCriteria.max_runtime_secs();
+    }
+
+    public int stopping_rounds() {
+      return _searchCriteria.stopping_rounds();
+    }
+
+    public ScoreKeeper.StoppingMetric stopping_metric() {
+      return _searchCriteria.stopping_metric();
+    }
+
+    public double stopping_tolerance() {
+      return _searchCriteria.stopping_tolerance();
+    }
+
+    public void set_seed(long seed) {
+      _searchCriteria.set_seed(seed);
+    }
+
+    public void set_max_models(int max_models) {
+      _searchCriteria.set_max_models(max_models);
+    }
+
+    public void set_max_runtime_secs(double max_runtime_secs) {
+      _searchCriteria.set_max_runtime_secs(max_runtime_secs);
+    }
+
+    public void set_stopping_rounds(int stopping_rounds) {
+      _searchCriteria.set_stopping_rounds(stopping_rounds);
+    }
+
+    public void set_stopping_metric(ScoreKeeper.StoppingMetric stopping_metric) {
+      _searchCriteria.set_stopping_metric(stopping_metric);
+    }
+
+    public void set_stopping_tolerance(double stopping_tolerance) {
+      _searchCriteria.set_stopping_tolerance(stopping_tolerance);
+    }
+
+    public void set_default_stopping_tolerance_for_frame(Frame frame) {
+      _searchCriteria.set_default_stopping_tolerance_for_frame(frame);
+    }
+
+    public static double default_stopping_tolerance_for_frame(Frame frame) {
```

Comment:
@seb-h2o I don't see usages and it is static. Maybe it is left after testing something.

Reply:
good catch, it's usage probably got lost during a conflict resolution -> fixed